package.json

```
@@ -34,6 +34,7 @@
     "test": "run-s test:all 'test:jest {@}' --",
     "test:all": "yarn run-all --no-sort test",
     "test:jest": "jest",
+    "spellcheck": "cspell packages/**/*.*",
```

The glob needs to be in quotation marks to work properly. I don't think we need to limit the spell-check to subfolders of packages -- let's set it loose on everything, and use the `ignorePaths` to turn off anything we don't want checked. I think the `--show-context` makes the output more useful too. The `--relative` shortens the file paths a bit, and once the glob was fixed and we have hundreds of files it became a bit tedious to have all the passing files listed to, so `--no-progress` suppresses that and we just see the errors.

```suggestion
    "spellcheck": "cspell lint --relative --no-progress --show-context '**/*'",
```